plugins/doc_fragments/auth.py

```diff
@@ -245,4 +253,22 @@ class ModuleDocFragment(object):
         ini:
           - section: hashi_vault_collection
             key: cert_auth_private_key
+      kubernetes_token:
+        env:
+          - name: ANSIBLE_HASHI_VAULT_KUBERNETES_TOKEN
+            version_added: 2.3.0
+        vars:
+          - name: ansible_hashi_vault_kubernetes_token
+            version_added: 2.3.0
+      kubernetes_token_path:
+        env:
+          - name: ANSIBLE_HASHI_VAULT_KUBERNETES_TOKEN_PATH
+            version_added: 2.3.0
+        ini:
+          - section: hashi_vault_collection
+            key: kubernetes_token_path
+            version_added: 2.3.0
+        vars:
+          - name: ansible_hashi_vault_kubernetes_token_path
+            version_added: 2.3.0
```

Comment:
We don't need `version_added` within these unless they were added after the option itself, so we can just put that annotation right on the whole option (see previous comment)

```suggestion
      kubernetes_token:
        env:
          - name: ANSIBLE_HASHI_VAULT_KUBERNETES_TOKEN
        vars:
          - name: ansible_hashi_vault_kubernetes_token
      kubernetes_token_path:
        env:
          - name: ANSIBLE_HASHI_VAULT_KUBERNETES_TOKEN_PATH
        ini:
          - section: hashi_vault_collection
            key: kubernetes_token_path
        vars:
          - name: ansible_hashi_vault_kubernetes_token_path
```